app/pulp/app/viewsets/base.py

```diff
@@ -87,11 +106,9 @@ def register_with(cls, router):
                        'correctly subclass the Master ViewSet, and do both have endpoint_name '
                        'set to different values?').format(cls.__name__)
                 warnings.warn(msg, RuntimeWarning)
```

Comment:
Yeah that makes sense. I don't see a benefit to letting Pulp be successful with missing endpoints. We want to make sure a problem like that fails obviously.